services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -2370,29 +2373,33 @@ public void updateStore(String clusterName, String storeName, UpdateStoreQueryPa
         hybridStoreConfigRecord.dataReplicationPolicy = hybridStoreConfig.getDataReplicationPolicy().getValue();
         hybridStoreConfigRecord.bufferReplayPolicy = hybridStoreConfig.getBufferReplayPolicy().getValue();
         setStore.hybridStoreConfig = hybridStoreConfigRecord;
-        if (!currStore.isHybrid() && setStore.partitionNum == 0) {
-          // This is a new hybrid store. Update store partition count from 0 to default value.
-          setStore.partitionNum = getVeniceHelixAdmin().getHelixVeniceClusterResources(clusterName)
-              .getConfig()
-              .getNumberOfPartitionForHybrid();
-          updatedConfigsList.add(PARTITION_COUNT);
-        }
       }
 
       if (incrementalPushEnabled.orElse(currStore.isIncrementalPushEnabled())
           && !veniceHelixAdmin.isHybrid(currStore.getHybridStoreConfig())
           && !veniceHelixAdmin.isHybrid(hybridStoreConfig)) {
         LOGGER.info(
-            "Enabling incremental push for a batch store:{} will convert it to a hybrid store with default configs.",
+            "Enabling incremental push for a batch store:{}. Converting it to a hybrid store with default configs.",
             storeName);
+        HybridStoreConfigRecord hybridStoreConfigRecord = new HybridStoreConfigRecord();
+        hybridStoreConfigRecord.rewindTimeInSeconds = DEFAULT_REWIND_TIME_IN_SECONDS;
+        updatedConfigsList.add(REWIND_TIME_IN_SECONDS);
+        hybridStoreConfigRecord.offsetLagThresholdToGoOnline = DEFAULT_HYBRID_OFFSET_LAG_THRESHOLD;
+        updatedConfigsList.add(OFFSET_LAG_TO_GO_ONLINE);
+        hybridStoreConfigRecord.producerTimestampLagThresholdToGoOnlineInSeconds = DEFAULT_HYBRID_TIME_LAG_THRESHOLD;
+        updatedConfigsList.add(TIME_LAG_TO_GO_ONLINE);
+        hybridStoreConfigRecord.dataReplicationPolicy = DataReplicationPolicy.NONE.getValue();
```

Comment:
Shall we use AA by default if write-compute is not enabled?

Reply:
We could not enable AA by default right now due to hybrid leader offset lag edge case. Remember we've discussed it before, because of the edge case, we also postpone enabling AA for hybrid stores in nuage.

Reply:
If that is the case, can we make the default mode configurable?

Reply:
AA for hybrid is already configurable: enable.active.active.replication.as.default.for.hybrid.store

Reply:
SG, thanks!